### PR TITLE
Add capability to update existing PR comment

### PR DESCRIPTION
**Pull request by rbelling**

### CODE REVIEW

**Comment by rbelling** [comment]
### TOTALS
| name | feat/upsert-comment | master |
| --- | --- | --- |
| js | 14.19KB (🔺 +1.31KB) | 12.87KB |

### FILE BREAKDOWN
| extension | feat/upsert-comment | master |
| --- | --- | --- |
| /build/lib/utils.js | 6.29KB (🔺 +1.31KB) | 4.98KB |
| /build/index.js | 149B | 149B |
| /build/commands/compare.js | 2.77KB | 2.77KB |
| /build/lib/index.js | 4.97KB | 4.97KB |

a watermark that lets us identify comments posted by bundle-checker

**Comment by rbelling** [comment]
### TOTALS
| name | feat/upsert-comment | master |
| --- | --- | --- |
| js | 14.14KB (🔺 +1.27KB) | 12.87KB |

### FILE BREAKDOWN
| extension | feat/upsert-comment | master |
| --- | --- | --- |
| /build/lib/utils.js | 6.25KB (🔺 +1.27KB) | 4.98KB |
| /build/index.js | 149B | 149B |
| /build/commands/compare.js | 2.77KB | 2.77KB |
| /build/lib/index.js | 4.97KB | 4.97KB |

**Comment by rbelling** [comment]
### TOTALS
| name | feat/upsert-comment | master |
| --- | --- | --- |
| js | 13.78KB (🔺 +929B) | 12.87KB |

### FILE BREAKDOWN
| extension | feat/upsert-comment | master |
| --- | --- | --- |
| /build/lib/utils.js | 5.89KB (🔺 +929B) | 4.98KB |
| /build/index.js | 149B | 149B |
| /build/commands/compare.js | 2.77KB | 2.77KB |
| /build/lib/index.js | 4.97KB | 4.97KB |

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-  - '11'
+  - "11"
 script:
   - yarn lint
   - yarn test
-  - yarn pack
-  - "./bin/run compare --prComment --gitRepository='https://github.com/rbelling/bundle-checker.git'
-    --installScript='yarn' --buildScript='yarn build' --currentBranch=$TRAVIS_PULL_REQUEST_BRANCH
-    --distPath='build' --targetBranch=$TRAVIS_BRANCH"
+  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then yarn pack; fi'
```
Comment (FeliceGeracitano) [inline .travis.yml]: Disable build & comment on master merge

File: src/lib/utils.ts [hunk 2]
```diff
@@ -17,6 +18,9 @@ const SHARED_TABLE_VALUES = {
   TOTALS_TITLE: 'TOTALS'
 };
 
+const COMMENT_WATERMARK =
+  '<p align="right">(Generate by <a href="https://github.com/rbelling/bundle-checker">bundle-checker 🔎📦</a>)</p>';
```
Comment (FeliceGeracitano) [inline src/lib/utils.ts]: I think looks nice => 🔎📦

**Comment by rbelling** [comment]
### TOTALS
| name | feat/upsert-comment | master |
| --- | --- | --- |
| js | 13.85KB (🔺 +1004B) | 12.87KB |

### FILE BREAKDOWN
| extension | feat/upsert-comment | master |
| --- | --- | --- |
| /build/lib/utils.js | 5.96KB (🔺 +1004B) | 4.98KB |
| /build/index.js | 149B | 149B |
| /build/commands/compare.js | 2.77KB | 2.77KB |
| /build/lib/index.js | 4.97KB | 4.97KB |

<p align="right">(Generate by <a href="https://github.com/rbelling/bundle-checker">bundle-checker 🔎📦</a>)</p>

**Pull request closed without merging**